EvalData/models/pairwise_assessment.py

```diff
@@ -110,8 +111,8 @@ def target_texts_with_diffs(self):
         if not self.target1Text or not self.target2Text:
             return (self.target1Text, self.target2Text)
 
-        toks1 = self.target1Text.split()
-        toks2 = self.target2Text.split()
+        toks1 = escape(self.target1Text).split()
+        toks2 = escape(self.target2Text).split()
```

Comment:
Please add to the method's docstring that it escapes HTML tags automatically.

Reply:
Added [https://github.com/AppraiseDev/Appraise/pull/56/commits/c1b2c6353bb9a6b881cfeab76610610a8ad879fc](url)

Reply:
Locally tested:
![local_test](https://user-images.githubusercontent.com/2372235/151082408-da9c1377-aaac-4767-939c-90ffe2b6f42d.png)